update.sh

```diff
@@ -58,11 +60,14 @@ for to_arch in $to_archs; do
 FROM scratch
 COPY qemu-${to_arch}-static /usr/bin/
 EOF
-        docker build -t ${DOCKER_REPO}:$from_arch-$to_arch ${work_dir}
+        docker build -t ${DOCKER_REPO}:$from_arch-$to_arch-${TAG_VER} ${work_dir}
+        docker tag ${DOCKER_REPO}:$from_arch-$to_arch-${TAG_VER} ${DOCKER_REPO}:$from_arch-$to_arch
         docker tag ${DOCKER_REPO}:$from_arch-$to_arch ${DOCKER_REPO}:$to_arch
         rm -rf "${work_dir}"
     fi
 done
 
-docker build -t ${DOCKER_REPO}:latest "${out_dir}/latest"
-docker build -t ${DOCKER_REPO}:register "${out_dir}/register"
+docker build -t ${DOCKER_REPO}:${TAG_VER} "${out_dir}/latest"
+docker tag ${DOCKER_REPO}:${TAG_VER} ${DOCKER_REPO}:latest
+docker build -t ${DOCKER_REPO}:register-${TAG_VER} "${out_dir}/register"
+docker tag ${DOCKER_REPO}:register-${TAG_VER} ${DOCKER_REPO}:register
```

Comment:
I think we do not need to do versioning for register image. I want to remove my modification here.
Because actually what `register` image is doing is just to execute the latest qemu's `qemu-binfmt-conf.sh` script [1] to create binfmt_misc files. It's not related to QEMU's version. `qemu-$arch-static` binary is used at user's run time, the timing to run arch specific binary.

Do you agree?

[1] https://github.com/multiarch/qemu-user-static/blob/master/containers/latest/Dockerfile#L4

Reply:
I removed the modification for now.